.github/workflows/release-automation.yaml

```diff
@@ -22,12 +22,16 @@ jobs:
         uses: "WyriHaximus/github-action-get-previous-tag@04e8485ecb6487243907e330d522ff60f02283ce" # v1.4.0
   generate-artifacts:
     needs: get-latest-tag
-    runs-on: ubuntu-20.04
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
I believe we were previously using ubuntu-20.04 for the dynamic binaries for a particular reason, if you check the PR comments for the original review

Reply:
i see older OS compatibility is the ask here.